public/app/features/serviceaccounts/ServiceAccountPage.tsx

```diff
@@ -199,6 +205,7 @@ export const ServiceAccountPageUnconnected = ({
               tokenActionsDisabled={tokenActionsDisabled}
             />
           )}
+          {canReadPermissions && <ServiceAccountPermissions serviceAccount={serviceAccount} />}
```

Comment:
Tests are failing due to this change. `<Permissions>` component makes some API calls, so it should be mocked to make tests passing. But if you don't need to test this component, you simply can change `hasAccessInMetadata()` mock in test file:
```ts
hasAccessInMetadata: () => false,
```